tests/functional/cleanup

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+. ~/test-projects/test1-openrc
+
+test_offer_uuid=$1
+
+echo "OFFER DELETE TEST"
+. "$ESI_TEST_CREDENTIALS/test1-openrc"
```

Comment:
I think @larsks means $ESI_TEST_CREDENTIALS would be the path to rc file.  For example,  set ```ESI_TEST_CREDENTIALS = ~/test-projects/test2-openrc``` at the beginning, then ```. ${ESI_TEST_CREDENTIALS:-./test1-openrc}```, it will source test2 project. But if you don't set ESI_TEST_CREDENTIALS at first, it will default to test1-openrc.

Reply:
I understood that, but I need to use different test projects in the same script, so I can't use one variable for all of them.